docs/topics/downloader-middleware.rst

```diff
@@ -429,6 +429,64 @@ In order to use this policy, set:
 * :setting:`HTTPCACHE_POLICY` to ``scrapy.extensions.httpcache.RFC2616Policy``
 
 
+.. _httpcache-policy-custom:
+
+Writing your own policy
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can implement a cache policy by creating a Python class that
+defines the methods described below.
+
+.. module:: scrapy.extensions.httpcache
+
+.. class:: CachePolicy
+
+    .. method:: should_cache_request(request)
+
+      Returns whether the request is cacheable or not
```

Comment:
Instead of ‘whether’, I think using ``True`` and ``False`` in the description may be more effective.

Also, see https://github.com/scrapy/scrapy/pull/3671#discussion_r268213806

And I personally feel like in many cases documenting parameters separately is overkill. For example, if the word ‘request’ here links to the API documentation of the `Request` class, I think we could remove the parameter-specific documentation lines below. @kmike What do you think?